app/src/main/java/com/hedvig/app/feature/birthday/viewmodel/BirthdayViewModel.kt

```diff
@@ -0,0 +1,59 @@
+package com.hedvig.app.feature.birthday.viewmodel
+
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import com.hedvig.app.feature.birthday.data.BirthdayRepository
+import io.reactivex.disposables.CompositeDisposable
+import io.reactivex.rxkotlin.plusAssign
+import timber.log.Timber
+import java.util.*
+
+
+class BirthdayViewModel(
+    private val birthdayRepository: BirthdayRepository
+) : ViewModel() {
+
+    private lateinit var birthDate: String
+
+    val isBirthdayData: MutableLiveData<Boolean> = MutableLiveData()
+    private val disposables = CompositeDisposable()
+
+
+
+    private fun isBirthday(): Boolean {
+
+        val birthMonth = birthDate.substring(4, birthDate.length - 6).toInt()
```

Comment:
Since `birthDate.length` always should be 12 this could be set to 6?

Reply:
I think this would be wise 🙂